src/Http/Middleware/SsoLogger.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Northwestern\SysDev\SOA\Http\Middleware;
+
+use http\ENV;
+use Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode as Middleware;
+use Auth;
+use App\Access;
+use Closure;
+use Illuminate\Support\Facades\Log;
+
+class SsoLogger extends Middleware
+{
+    protected $max_logs;
+
+    public function __construct() { $this->max_logs = ENV('SSO_LOG_MAX_ENTRIES',1000000); }
+
+    public function handle($request, Closure $next, ...$guards)
+    {
+        $controller = $request->route()->controller;
+
+        if (class_basename($controller) == "WebSSOController") {
```

Comment:
I think just the contents of the `else` block are necessary?

My concern here would be somebody remaining the controller. Once the make:websso commands ejects it, laravel-soa has no control over what people do to it, so they could rename it `MyCoolWebSSOControllerForTheWebSSOUpgrade` and miss this behaviour.